packages/ketcher-react/src/script/editor/operations/descriptors.ts

```diff
@@ -0,0 +1,78 @@
+/****************************************************************************
+ * Copyright 2020 EPAM Systems
```

Comment:
2021

Reply:
fixed in [update copyrights](https://github.com/epam/ketcher/pull/331/commits/75d9332884c442c41a205b5bc673efae70972879)